inbox/email_J_11.md

```diff
@@ -0,0 +1,8 @@
+Hi Team,
+
+Our users have started sending in requests to understand whether the movie is rated G, PG..... and they also want some more information on how long the movie is. Could you take a look and see if it is possible to display these details?
```

Comment:
please can we make this "..." instead of "....." ? and also capitalize and.

sorry 😂

Reply:
of course check now XD